### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by pompompurins** [psauxx/dbin#10]
Hi, I need to use this code in production, is it still active // is it safe to use?

**Comment by AndroTwrpid** [comment]
I think yeah, is 4 months.

**Comment by psauxx** [comment]
u know php, i say strip the parts needed and the rest is crap code. wish i couldv done better

**Comment by AndroTwrpid** [comment]
Yeah also the CSS is managed in a terible method: doxbin.org is down right now and the css + js are not working neither. I was able to recover some css files but not all. I will update you when i'am able to recover all files.

**Comment by psauxx** [comment]
That's true u can go to doxbin.com and org css there they moved domains
